datadog_checks_dev/tests/tooling/manifest_validator/input_constants.py

```diff
@@ -196,9 +196,11 @@
 
 DISPLAY_ON_PUBLIC_VALID_MANIFEST = JSONDict({"display_on_public_website": True})
 
-INVALID_DIFF = ['app_id', 'id']
+INVALID_DIFF = 'app_id id'
```

Comment:
Does changing the type of this and `VALID_DIFF` from list to a string have any side effects?

Reply:
No, the expected output is actually a string (the git diff). This way, the regex will not throw errors when trying to match.